package com.ritense.valtimoplugins.valuemapper.exception

class ValueMapperCommandException(message: String? = null, cause: Throwable? = null) : RuntimeException(message, cause)
